src/app/app.module.ts

```diff
@@ -14,6 +14,10 @@ import { UsersContainerComponent } from './components/users-container/users-cont
 import { UsersListComponent } from './components/users-container/users-list/users-list.component';
 import { UsersDataStore } from './store/users-data.store';
 import { SearchBarComponent } from './components/search-bar/search-bar.component';
+import { ListModule } from 'patternfly-ng';
```

Comment:
You can merge all the imports into one line from `patternfly-ng`

Reply:
okay!

Reply:
sure